site/cypress/e2e/pronouns_change.spec.js

```diff
@@ -0,0 +1,75 @@
+
+describe('Tests cases abut changing user pronouns', () => {
+
+    let oldPronouns = '';
+
+    it('Tests changing pronouns for student', () => {
+        cy.visit('/user_profile');
+        cy.login('student');
+
+        //open pronouns form
+        cy.get('#pronouns_val').click();
+        const e = cy.get('#user-pronouns-change');
+
+        //save old pronouns
+        e.then(($pronounsInput) => {
+            oldPronouns = $pronounsInput.val();
+        });
+
+        //type in new pronouns
+        cy.get('button[aria-label="Clear pronoun input"]').click(); //clear input using trash can
+        e.type('They/Them');
+        cy.get('#edit-pronouns-form .form-button-container > .btn-primary').first().click();
```

Comment:
Can we target this button in a more specific way?  It's very possible that someone will add another primary button to this form and thus break the test.  It would be better to give the button an ID, or otherwise target it in a more unique way.

Reply:
done